### PR TITLE
Updates to FAQ for Intel SGX IAS v2 and Status Code 429

### PR DESCRIPTION
**Pull request by danintel**
- Add Status Code 429 Error Information . I am adding this question so if in the future if anyone has the
same situation he/she can directly find the root cause.

Signed-off-by: danintel <daniel.anderson@intel.com> on behalf of yashvadhvani <yashvadhvani@gmail.com>

- Add answer for using Intel SGX IAS API version 2

- Fix spelling error

Signed-off-by: danintel <daniel.anderson@intel.com>

### CODE REVIEW

File: generator/source/faq/consensus.rst [hunk 1]
```diff
@@ -265,6 +265,13 @@ In PoET, forks occur due to a network partition, the size of the network, the ti
 
 TPs don’t really affect forks, unless they have a severe impact on the validation duration of the block. However, unresolvable forks due to non-determinism, are likely a TP problem.
 
+How do I fix PoET using Intel SGX IAS API version 2, which is end of life?
+--------------------------------------------------------------------------
+For those who are trying to use PoET SGX with IAS, you need to move to the IAS API v3 interface. Basically just go through and change the IAS URLs from `/v2/` to `/v3/`
+The change needs to be made in files `ias_client/sawtooth_ias_client/ias_client.py`
+and `ias_client/tests/unit/test_ias_client.py` .
```
Comment (dcmiddle) [inline generator/source/faq/consensus.rst:272]: Can you also include a link to the v3 API:
https://software.intel.com/sites/default/files/managed/7e/3b/ias-api-spec.pdf

Comment (danintel) [reply generator/source/faq/consensus.rst:272]: Yes.

File: generator/source/faq/rest.rst [hunk 2]
```diff
@@ -110,6 +110,10 @@ I am getting this error: ``Cross-Origin Request Blocked: The Same Origin Policy
 ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 The Sawtooth REST API doesn't support CORS. To allow cross-origin access to the Sawtooth API, put it behind a proxy.
 
+What does this error mean: ``Request failed with status code 429``
+------------------------------------------------------------------
+To avoid DDoS attacks (too many requests from a single source), Sawtooth has a mechanism called "backpressure test" which avoids such things as excessive network traffice and excessive Sawtooth transactions.
```
Comment (annechenette) [inline generator/source/faq/rest.rst]: traffice -> traffic

**Review by annechenette — CHANGES_REQUESTED** [review]

**Review by annechenette — APPROVED** [review]

File: generator/source/faq/rest.rst (hunk 2 above)
Comment (danintel) [reply generator/source/faq/rest.rst]: Fixed.

**Review by dcmiddle — APPROVED** [review]

**Pull request merged**